features/async-clipboard.yml.dist

```diff
@@ -56,6 +56,23 @@ compat_features:
   #   safari_ios: "13.4"
   - api.ClipboardItem.ClipboardItem
 
+  # baseline: false
```

Comment:
This is wrong, because https://github.com/mdn/browser-compat-data/pull/23593 hasn't reached us through a BCD release yet. Let's wait for that and not publish an incorrect status in the interim.

Reply:
Fixed with e322f9f4e934960d5f7a9fd7cef2f33f8c44b036 and 9d9776682d58898f8fe81f90fd9821ef9c72ab15.